apps/package.json

```diff
@@ -139,7 +139,7 @@
     "node-sass": "^4.0.0",
     "npm-which": "2.0.0",
     "phantomjs-prebuilt": "^2.1.3",
-    "playground-io": "0.4.0",
+    "playground-io": "code-dot-org/playground-io",
```

Comment:
I've [published release info](https://github.com/code-dot-org/playground-io/releases) (which created tags) for both Code.org-specific versions on [code-dot-org/playground-io](https://github.com/code-dot-org/playground-io).  I also just pushed a commit here pinning playground-io to tag v0.4.0-cdo.1.